src/data/chbp_data_2019.json

```diff
@@ -380,40 +488,44 @@
           [
             [
               [
-                -122.319617008927892,
-                47.614020740733515
+                -122.31961700893,
+                47.61402074073
               ],
               [
-                -122.319616878569974,
```

Comment:
There’s a bunch of small changes like this, are these just shifting the features around?

Reply:
I did some geometry cleanup on existing geometry while digitizing Chophouse Row to square off some corners to improve the look at this scale. The small movements are just the corners being shifted slightly.